tests/datm_conf/test_case_cfsr.IN

```diff
@@ -0,0 +1,25 @@
+mkdir INPUT DATM_INPUT RESTART history MOM6_OUTPUT
+
+# FV3 fixed input
+cp    @[RTPWD]/DATM/cfsr.SCRIP.nc DATM_INPUT
+export IATM=1760
+export JATM=880
+
+# MOM6 fixed input
+cp    @[RTPWD]/MOM6_FIX_025deg/* ./INPUT
+
+# CICE5 fixed input
+cp    @[RTPWD]/CICE_FIX_mx025/grid_cice_NEMS_mx025.nc .
+cp    @[RTPWD]/CICE_FIX_mx025/kmtu_cice_NEMS_mx025.nc .
+
+# ICs
+ln -s @[RTPWD]/DATM/CFSR/201110/cfsr*nc DATM_INPUT/
```

Comment:
Why are you linking the november and december input files in addition to the october files?

Reply:
These two optional runs are currently set up to run for 20 days. But the same setup can be used to integrate the mode for up to 3 months. The data from November and December input files are there so that the scripts do not need to be updated when I add instructions to make longer (3 months) runs on the wiki page soon.

For MOM6 and CICE5, only initial conditions are needed for making longer runs. But for DATM, time-dependent forcing data are needed. For CFSR forcing, input files from December 2011 are the most recent ones that have been processed.